spec/inputs/file_spec.rb

```diff
@@ -165,4 +166,14 @@
     insist { events[1]["path"] } == "#{tmpfile_path}"
     insist { events[1]["host"] } == "#{Socket.gethostname.force_encoding(Encoding::UTF_8)}"
   end
+
+  context "sincedb_path is an existing directory" do
+    it "should raise exception" do
+      tmpfile_path = Stud::Temporary.pathname
+      Stud::Temporary.directory do |sincedb_path|
+        plugin = LogStash::Inputs::File.new("path" => tmpfile_path, "sincedb_path" => sincedb_path)
```

Comment:
I would rather move this to be a subject or a let variable as is the core of this context/describe section of the test.